.github/workflows/audit-rust.yml

```diff
@@ -28,6 +28,15 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions-rs/audit-check@v1
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Download cargo-audit
+        run: |
+          set +f
+          curl -s https://api.github.com/repos/rustsec/rustsec/releases/latest | grep "browser_download_url.*cargo-audit-x86_64-unknown-linux-musl.*" | cut -d : -f 2,3 | tr -d \" | wget -qi -
+          tar -xvzf ./cargo-audit*.tgz
+          mv ./cargo-audit*/cargo-audit ./cargo-audit
+          chmod +x ./cargo-audit
+          set -f
```

Comment:
Also targeting `/latest` and by extension not having checksums or the like, yeah. That's waiting for flakes. :sweat_smile:

Reply:
i apparently forgot that caching exists...

Also i didn't know that audit-check does nothing else than `cargo install cargo-audit` - i thought it does something more fancy - will do that too then, unless we switch to actions-rust-lang (see my other comment) or if we want to create our own fork - an action is probably good because of the issue it can create.

Reply:
Well the caching of it, so it doesn't cargo install when cache is available, is what I think the fancy part is :laughing:
As well as version pinning I suppose.